main.js

```diff
@@ -0,0 +1,111 @@
+const game = {
+    board: [[5, 4, 3, 2, 1], [], []],
+    peg1EmptiedOnce: false,  // Track whether peg1 has been emptied
+
+    // Display the current state of the board using map
+    displayBoard() {
+        console.log("--- Board State ---");
+        this.board.map(peg => {
+            console.log("--- " + peg.join(' '));
+        });
+    },
+
+    // Move a disc from one peg to another
+    moveDisc(fromPegNumber, toPegNumber) {
+        const fromPegIndex = fromPegNumber - 1; // Convert peg number to index (1 -> 0, 2 -> 1, etc.)
+        const toPegIndex = toPegNumber - 1;     // Convert peg number to index
```

Comment:
DRY => don't repeat yourself (you'll be reading this a lot from me =)) 
extract this logic to an util function and reuse, don't write twice the same code.
```suggestion
        const fromPegIndex = convertPegIndexToNumber(fromPegNumber) 
        const toPegIndex =  convertPegIndexToNumber(toPegNumber) 
```